students/shawn_degraw/session02/fizz_buzz.py

```diff
@@ -0,0 +1,12 @@
+#fizz buzz function
+for i in range(1,101):
+    result3 = i % 3
```

Comment:
It is not necessary to define variables and assign the results of a simple math operation to use these results in if statements.